packages/salesforcedx-vscode-apex/src/languageClientUtils.ts

```diff
@@ -17,6 +17,7 @@ import {
 
 export class LanguageClientUtils {
   private static instance: LanguageClient | undefined;
+  public static indexing = true;
```

Comment:
This is to prevent the refresh command from running before the lsp is finished indexing the project. Otherwise you'll get a message such as "no tests found" which is more confusing than "Loading tests". Another idea I had was maybe hiding the test bar icon until this process is complete and when it's ready show the icon. Thoughts? @ntotten